### PR TITLE
[WIP] UX Guidelines for Firefox for Android extensions 

### PR DESCRIPTION
**Pull request by caitmuenster**
Creates a page for Bram's UX guidelines for extensions for Firefox for Android to the site. This is still a draft. :) 

(@muffinresearch, I wasn't able to see the table of contents when I looked at the page on local, and I have no idea why it isn't showing up! Could you help me look into it?)

### CODE REVIEW

File: src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+# Introduction
+
+::: note alert
+In August 2020, Mozilla launched a new, reimagined Firefox for Android experience (codenamed "Fenix"). The  browser for Android has been rebuilt from the ground up using [GeckoView](https://mozilla.github.io/geckoview/), Mozilla's mobile browsing engine.
+
+Currently, only a limited number of [Recommended Extensions](https://support.mozilla.org/kb/recommended-extensions-program?utm_source=extensionworkshop.com&utm_medium=dev-article&utm_content=developing-extensions-for-firefox-for-android) are supported. However, we are continuously working on increasing support, taking into account usage and feedback to ensure we are making the most of our available resources. We will post updates to [The Add-ons Blog](https://blog.mozilla.org/addons/category/mobile?utm_source=extensionworkshop.com&utm_medium=dev-article&utm_content=differences-between-desktop-and-android-extensions) as plans solidify each quarter.
```
Comment (kewisch) [inline src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md:39]:
```suggestion
Currently, only a limited number of [Recommended Extensions](https://support.mozilla.org/kb/recommended-extensions-program?utm_source=extensionworkshop.com&utm_medium=dev-article&utm_content=developing-extensions-for-firefox-for-android) are supported. However, we are continuously working on increasing support, taking into account usage and feedback to ensure we are making the most of our available resources. We will post updates to [the Add-ons Blog](https://blog.mozilla.org/addons/category/mobile?utm_source=extensionworkshop.com&utm_medium=dev-article&utm_content=differences-between-desktop-and-android-extensions) as plans solidify each quarter.
```

File: src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# User Experience Guidelines for Mobile Extensions
+
+TEST TEST TEST
+
+{% endcapture %}
```
Comment (kewisch) [inline src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md:23]:
```suggestion

{% endcapture %}
```

File: src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+<!-- END: Page Hero Banner -->
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+# Introduction
+
+::: note alert
+In August 2020, Mozilla launched a new, reimagined Firefox for Android experience (codenamed "Fenix"). The  browser for Android has been rebuilt from the ground up using [GeckoView](https://mozilla.github.io/geckoview/), Mozilla's mobile browsing engine.
```
Comment (kewisch) [inline src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md:37]: Maybe want to link https://blog.mozilla.org/blog/2020/08/25/introducing-a-new-firefox-for-android-experience/ ?

File: src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+# The basics
+Make your extension’s stylesheet responsive and follow these key guidelines to ensure that your extension works well for as many people as possible on a variety of devices.
+
+## Color contrast
+
+Pass your colors through WebAIM’s [Color Contrast Checker](https://webaim.org/resources/contrastchecker/). They should pass WCAG AA level.
+
+## Layout
+
+- Optimise for a 360 × 640 dp layout
```
Comment (kewisch) [inline src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md:67]:
```suggestion
- Optimise for a 360 × 640 dp layout.
```

**Review by kewisch — APPROVED** [review]
Looks good to me, thanks for putting this together! A few comments to consider:

File: src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md (hunk 2 above)
Comment (caitmuenster) [reply src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md:23]: 🤦‍♀️

**Comment by muffinresearch** [comment]
> Creates a page for Bram's UX guidelines for extensions for Firefox for Android to the site. This is still a draft. :)
> 
> (@muffinresearch, I wasn't able to see the table of contents when I looked at the page on local, and I have no idea why it isn't showing up! Could you help me look into it?)

@caitmuenster Yep, I'm looking into it...

**Comment by muffinresearch** [comment]
@caitmuenster See the commit - this adds the config to enable the toc, and it also updates the heading levels since they were off by one level.

**Comment by muffinresearch** [comment]
One other suggestion might be to change the URL to expand ux to user-experience for consistency (we already have https://github.com/mozilla/extension-workshop/blob/master/src/content/documentation/develop/user-experience-best-practices.md)

**Comment by caitmuenster** [comment]
@muffinresearch Thank you so much -- both for the heading fixes and for fixing the TOC issue! My face went 😮 when I saw that  comma on pages.json. 

+1 to your idea about changing the page name.

**Pull request merged**